src/api/metadata/index.ts

```diff
@@ -0,0 +1,40 @@
+import log from 'loglevel'
+import { Context } from '/utils/context'
+import { SupportedChainId as ChainId } from '/constants/chains'
+import { getSerializedCID, loadIpfsFromCid } from '/utils/appData'
+import { AppDataDoc } from './types'
+import { CowError } from '/utils/common'
+
+export class MetadataApi<T extends ChainId> {
+  chainId: T
```

Comment:
The metadata related fns are chain agnostic.
I can't think of a reason why we would need to know which chain it's on other than maybe logging.

Remove maybe?